src/test/java/subway/StationAcceptanceTest.java

```diff
@@ -55,13 +56,72 @@ void createStation() {
      * When 지하철역 목록을 조회하면
      * Then 2개의 지하철역을 응답 받는다
      */
-    // TODO: 지하철역 목록 조회 인수 테스트 메서드 생성
+    @DisplayName("지하철역 목록을 조회한다.")
+    @Test
+    void selectStations(){
+        // Given
+        ExtractableResponse<Response> response = createStation("강남역");
+        assertThat(response.statusCode()).isEqualTo(HttpStatus.CREATED.value());
+
+        ExtractableResponse<Response> response2 =  createStation("서초역");
+        assertThat(response2.statusCode()).isEqualTo(HttpStatus.CREATED.value());
+
+        // When
+        List<String> stationNames = getStations();
+
+        // Then
+        assertThat(stationNames).contains("강남역", "서초역");
+    }
+
+    // 지하철역 이름으로 지하철역 생성
+    private ExtractableResponse<Response> createStation(String stationName){
+        StationRequest stationRequest = new StationRequest();
+        stationRequest.setName(stationName);
+
+        return RestAssured.given().log().all()
+                        .body(stationRequest)
+                        .contentType(MediaType.APPLICATION_JSON_VALUE)
+                        .when().post("/stations")
+                        .then().log().all()
+                        .extract();
+    }
+
+    // 모든 지하철 목록 조회
+    private List<String> getStations(){
+        return RestAssured.given().log().all()
+                .when().get("/stations")
+                .then().log().all()
+                .statusCode(HttpStatus.OK.value())
+                .extract()
+                .jsonPath().getList("name", String.class);
+    }
 
     /**
      * Given 지하철역을 생성하고
      * When 그 지하철역을 삭제하면
      * Then 그 지하철역 목록 조회 시 생성한 역을 찾을 수 없다
      */
-    // TODO: 지하철역 제거 인수 테스트 메서드 생성
+    @DisplayName("지하철역을 삭제한다.")
+    @Test
+    void deleteStations(){
+        // Given
+        ExtractableResponse<Response> response = createStation("서초역");
+        assertThat(response.statusCode()).isEqualTo(HttpStatus.CREATED.value());
 
+        long id = response.body().jsonPath().getLong("id");
+
+        ExtractableResponse<Response> response1 =
+                RestAssured.given().log().all()
+                        .contentType(MediaType.APPLICATION_JSON_VALUE)
+                        .when().delete("/stations/{id}", id)
+                        .then().log().all()
+                        .extract();
+        assertThat(response.statusCode()).isEqualTo(HttpStatus.CREATED.value());
```

Comment:
```suggestion
        assertThat(response1.statusCode()).isEqualTo(HttpStatus.NO_CONTENT.value());
```

삭제에 대한 응답 status 검증도 있으면 좋겠네요 😄 
변수명도 조금 더 의미를 부여하면 좋겠습니다 🔥